src/dump.c

```diff
@@ -1349,7 +1355,7 @@ static jl_value_t *jl_deserialize_value_array(jl_serializer_state *s, jl_value_t
     for (i = 0; i < ndims; i++) {
         dims[i] = jl_unbox_long(jl_deserialize_value(s, NULL));
     }
-    jl_array_t *a = jl_new_array_for_deserialization((jl_value_t*)NULL, ndims, dims, isunboxed, elsize);
+    jl_array_t *a = jl_new_array_for_deserialization((jl_value_t*)NULL, ndims, dims, isunboxed, elsize, elalign);
```

Comment:
@vtjnash Any idea how to get `elalign` at this point of the serialisation state? I don't have a eltype here yet, right? For that I need `aty` and that is depended on `a`

Reply:
compute it during serialization

Reply:
Ok just not keep it around ;) Gotcha